web/frontend/src/components/pcd/pcd-view.vue

```diff
@@ -509,80 +515,103 @@ watch(() => props.pointcloud, (updated?: Uint8Array) => {
 </script>
 
 <template>
```

Comment:
Everything that follows on this file is reorganization of HTML to somewhat un-break the 3d renderer's layout.

Reply:
I think it would be a cool exercise for the JS guild at some point to try to break this component up even further and split apart all HTML presentation layout from the 3D render setup

Reply:
Agreed! Added to guild action items